packages/code-du-travail-data/indexing/index.js

```diff
@@ -109,11 +110,50 @@ async function main() {
     index: `${CDTN_INDEX_NAME}-${ts}`,
     name: CDTN_INDEX_NAME
   });
+
+  const currentIndices = [
+    `${CDTN_INDEX_NAME}-${ts}`,
+    `${THEMES_INDEX_NAME}-${ts}`,
+    `${CDTN_CCN_NAME}-${ts}`,
+    `${ANNUAIRE_INDEX_NAME}-${ts}`
+  ];
+
+  function isCdtnIndex({ index }) {
+    const patterns = [
+      CDTN_INDEX_NAME,
+      THEMES_INDEX_NAME,
+      CDTN_CCN_NAME,
+      ANNUAIRE_INDEX_NAME
+    ];
+    return patterns.some(pattern => index.startsWith(`${pattern}-`));
+  }
+
+  const { body: indices } = await client.cat.indices({ format: "json" });
+
+  const pIndicesToDelete = indices
```

Comment:
this critical function should have some TUs

Reply:
ahah true ! I wonder if I should move that to a dedicated function

Reply:
sound good !